kiwmi/luak/kiwmi_keyboard.c

```diff
@@ -16,6 +16,29 @@
 #include "input/keyboard.h"
 #include "luak/kiwmi_lua_callback.h"
 
+static int
+l_kiwmi_keyboard_configure(lua_State *L)
```

Comment:
I don't feel like configure is a nice name for this. Something like change_keymap instead? Open to suggestions.

Reply:
yeah I renamed it to l_kiwmi_keyboard_set_keymap